acceptance/setup/aio/010_Install.rb

```diff
@@ -31,17 +30,20 @@
       on(master, "rpm -Uvh https://yum.puppetlabs.com/puppet5-release-el-#{version}.noarch.rpm")
     end
   else
-    if ENV['SERVER_VERSION']
-      install_puppetlabs_dev_repo(master, 'puppetserver', ENV['SERVER_VERSION'])
-      install_puppetlabs_dev_repo(master, 'puppet-agent', ENV['SHA'])
+    if ENV['SERVER_VERSION'] && dev_builds_url == 'http://builds.delivery.puppetlabs.net'
+      install_from_build_data_url('puppetserver', "#{dev_builds_url}/puppetserver/#{ENV['SERVER_VERSION']}/artifacts/#{ENV['SERVER_VERSION']}.yaml", master)
     else
-      # beaker can't install puppetserver from nightlies (BKR-673)
-      repo_configs_dir = 'repo-configs'
-      install_repos_on(master, 'puppetserver', 'nightly', repo_configs_dir)
-      install_repos_on(master, 'puppet-agent', ENV['SHA'], repo_configs_dir)
+      if ENV['SERVER_VERSION'].nil? || ENV['SERVER_VERSION'] == 'latest'
+        server_version = ''
+        project = 'puppetserver-latest'
+      else
+        server_version = ENV['SERVER_VERSION']
+        project = 'puppetserver'
+      end
+      install_puppetlabs_dev_repo(master, project, server_version, nil, :dev_builds_url => dev_builds_url)
```

Comment:
If `ENV['SERVER_VERSION']` isn't set, and `dev_builds_url` is set to `http://builds.delivery.puppetlabs.net`, this will fail

Reply:
Ah, that's fun.

Reply:
Do you want me to add in extra checks to catch that case?

Reply:
In Jenkins, we set SERVER_VERSION to `latest`. We should default to that in general if not set.

Reply:
That would also allow you to simplify the project variable above.